grunt/config/replace.js

```diff
@@ -84,25 +84,21 @@ module.exports = function (grunt, options) {
     dist: {
       options: {
         silent: true,
-        patterns: [
-          {
-            json: function (done) {
-              done(generatePatterns());
-            }
+        patterns: [{
+          json: function(done) {
+            done(generatePatterns());
           }
-        ]
+        }]
       },
-      files: [
-        {
-          expand: true,
-          flatten: true,
-          src: [
-            path.join(options.outputdir, '*.xml'),
-            path.join(options.outputdir, '*.html')
-          ],
-          dest: options.outputdir
-        }
-      ]
+      files: [{
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?